app/helpers/documents_overview_helper.rb

```diff
@@ -1,5 +1,5 @@
 module DocumentsOverviewHelper
-  def edit_document_path(document_type)
+  def edit_document_path_for(document_type)
```

Comment:
I had to change the name of this helper because the edit_document_path helper overwrote the default behavior of "edit_document_path" routing helper. Overwriting default helpers seems prone to odd behavior/bugs. It wasn't being used anywhere else, and I did need to use the default "edit_document_path" helper, so I changed the name of this helper.